website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -58,6 +58,7 @@ apply to both Consul Enterprise and Consul community edition (CE).
 
 | Consul Version | Compatible Envoy Versions              |
 | -------------- | -------------------------------------- |
+| 1.19.x CE      | 1.29.4, 1.28.3, 1.27.5, 1.26.8         |
 | 1.18.x CE      | 1.28.3, 1.27.5, 1.26.8, 1.25.11        |
```

Comment:
Maybe fine, but IIRC in the past we've waited to update this matrix until the corresponding release was out. We might want to ask @blake 's opinion here

Reply:
@zalimeni That's correct. This update should only be published once 1.19 is available.

Reply:
But since this is only going to `main`, it won't be available until after the release and [publish-website](https://github.com/hashicorp/consul/blob/943f0072c00b979dba3bceb19ebafff5bcfaa435/.release/ci.hcl#L123-L134) steps, correct?

Reply:
@DanStough that's right - sorry for not being clearer in my question @blake , as Dan said this is only on `main`, and would only be visible in the repo. I think it's fine to update here in advance of 1.19, given nothing will be _published_ until docs are out for that release. It definitely makes the pre-release steps simpler to do it this way.